docs/src/main/tut/datatypes/NonEmptyList.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: docs
+title:  "NonEmptyList"
+section: "data"
+source: "core/src/main/scala/cats/data/NonEmptyList.scala"
+scaladoc: "#cats.data.NonEmptyList"
+---
+# NonEmptyList
+
+If you have had the opportunity of taking a look to [Validated](validated.html) or [IOR](ior.html), you'll find that a common case is to use `NonEmptyList` with one of these data structures.
+
+Why? Because it fits nicely in the error reporting cases. As stated by its name, `NonEmptyList` is a _specialized_ data type that will have, at least, one element. You'll find that its behavior is like a list, with the aforementioned constraint. Think of it as a `List` wrapper.
```

Comment:
What does specialized mean here? When I first read this, I thought about `@specialized` but I guess it's not? I think this might be confusing.

Reply:
Sorry, I mean that is a _specific-purpose_ data type (compared to a regular `List`). I'll rephrase this.